apps/src/templates/rubrics/rubrics.module.scss

```diff
@@ -195,6 +195,18 @@ p.aiTokenText {
   color: $product_affirmative_default;
 }
 
+h2.studentName {
```

Comment:
This is some clean up work from an earlier PR to get styles out of the components and into scss.